containerd-http-proxy/configure_http_proxy.yaml

```diff
@@ -0,0 +1,94 @@
+kind: DaemonSet
```

Comment:
is there anything we need to add to whitelist this? Should we add a note here for the future reviews that even minor modifications of this yaml will result in the broken whitelist in Autopilot?

Reply:
Added a README with instructions as well.